code/_onclick/hud/action_button.dm

```diff
@@ -14,8 +15,9 @@
 		moved = 0
 		usr.update_action_buttons() //redraw buttons that are no longer considered "moved"
 		return 1
-	if(usr.next_move >= world.time) // Is this needed ?
```

Comment:
Next_move is clickdelay, people shouldn't be unable to use action buttons because they whacked someone.